applications/dashboard/models/class.tagmodel.php

```diff
@@ -530,6 +530,7 @@ public function getDiscussions($tag, $limit, $offset) {
             ->join('Tag t', 't.TagID = td.TagID')
             ->whereIn('t.Name', $tags)
             ->limit($limit, $offset)
+            ->orderBy('td.DiscussionID', 'desc')
```

Comment:
What about adding sort order and direction as parameters on the function, with these values as the defaults, instead of forcing a sort order? Maybe one day we'll want an alphabetized list, etc.

Reply:
That would be a better practice.